schema_editor/test/mock/resources/resources-mocks.js

```diff
@@ -3,25 +3,70 @@
     'use strict';
 
     function ResourcesMock () {
-        var RecordTypeResponse = {
+        var RecordSchemaResponse = {
             'count': 1,
             'next': null,
             'previous': null,
             'results': [
                 {
-                    'uuid': '362be38f-9110-4b8f-83a5-4280251483a1',
-                    'current_schema': '3ba9bc8f-7e4a-490e-8a14-666808711b10',
-                    'created': '2015-05-12T17:41:09.357521Z',
-                    'modified': '2015-05-12T17:41:09.357562Z',
-                    'label': 'Accident',
-                    'plural_label': 'Accidents',
-                    'description': 'An accident.',
-                    'active': true
+                    'uuid': '80c10057-2cfc-4a32-8e3c-0573e8bf8f3f',
+                    'schema': {
+                        // TODO: is this right? What are these blank properties for?
```

Comment:
Yeah this is ok for now. I was temporarily saving each of the related content types generated via the admin interface in the definitions block, although eventually they won't live there, but instead will need to be serialized/de-serialized from this schema object as was discussed yesterday. Not totally sure what this looks like yet.